app/views/layout.nunjucks

```diff
@@ -10,5 +10,16 @@
   </head>
   <body>
     {% block content %}{% endblock %}
+
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+      ga('create', 'UA-67365892-5', 'auto');
```

Comment:
Let's move this to an environment variable to it can be changed per environment. And wrap it in a conditional block.